libs/test/SalesforceSDKTest/res/values/bootconfig.xml

```diff
@@ -6,6 +6,7 @@
     <string-array name="oauthScopes">
         <item>api</item>
         <item>web</item>
+        <item>openid</item>
```

Comment:
Adding `openid` scope to the test project.

Reply:
@rwhitley I assume this will change documentation in a few places.

Reply:
@brandonpage Not really, no. It doesn't change existing documentation. This scope is only required for apps that need `openid`. It will be a part of the new OpenID token documentation.